src/components/utils/popover/components/popover-item/popover-item.types.ts

```diff
@@ -1,7 +1,23 @@
+
+/**
+ * Represents popover item delimiter
```

Comment:
Describe what is "popover item delimiter"

Reply:
added